src/index/diskann/diskann_config.h

```diff
@@ -33,13 +34,6 @@ class DiskANNConfig : public BaseConfig {
     // This is the flag to enable fast build, in which we will not build vamana graph by full 2 round. This can
     // accelerate index build ~30% with an ~1% recall regression.
     CFG_BOOL accelerate_build;
-    // The number of threads used for preparing and searching. When 'num_threads' uses as build parameter, the indexing
-    // time improves almost linearly with the number of threads (subject to the cores available on the machine and DRAM
-    // bandwidth). When 'num_threads' uses as prepare parameter, Threads run in parallel and one thread handles one
-    // query at a time. More threads will result in higher aggregate query throughput, but will also use more IOs/second
-    // across the system, which may lead to higher per-query latency. So find the balance depending on the maximum
-    // number of IOPs supported by the SSD.
-    CFG_INT num_threads;
```

Comment:
num_threads should not delete, it is needed by for_train() function.

Reply:
updated